Makefile

```diff
@@ -176,8 +177,8 @@ deploy-once:
 
 
 copy-to-s3: guard-path
-	@for cfn in $(CFN_TEMPLATES) ; do \
-		aws s3 cp $${cfn} s3://$(BUCKET)/$(path)/$${cfn} ; \
+	@for key in $(CFN_TEMPLATES) $(IAM_POLICIES) ; do \
+		aws s3 cp $${key} s3://$(BUCKET)/$(path)/$${key} ; \
```

Comment:
how is the $(BUCKET) var and others getting populated, where do we expect to write the data and what permissions are required?

Reply:
`BUCKET` defaults to `cz-provision-account` and can be overridden at the CLI. `CFN_TEMPLATES` and `IAM_POLICIES` are make variables that dynamically contain all of the templates and policies; `path` is a dynamically composed version.